services/src/main/java/org/exoplatform/news/service/impl/NewsTargetingServiceImpl.java

```diff
@@ -115,7 +115,7 @@ public void saveNewsTarget(String newsId, boolean staged, List<String> targets,
   }
 
   @Override
-  public List<MetadataItem> getNewsTargetItemsByTargetName(String targetName, long offset, long limit) {
+  public List<MetadataItem> getNewsTargetItemsByTargetName(String targetName,boolean isArchived, long offset, long limit) {
```

Comment:
Does the isArchived parameter is used ?

Reply:
I will check, many changes will be made, I will remove it after checking.

Reply:
if you PR is still in progress, perhaps you can mark it as draft ?